tests/integration/targets/win_availability_group/tasks/main.yml

```diff
@@ -11,6 +11,7 @@
     use_last_backup: false
     healthcheck_timeout: 15000
     basic_availability_group: false
+    contained_availability_group: false
```

Comment:
Can you also add a test where this is `true`?

Reply:
New test in tests/inegration? I added some tasks in the win_availability_group that run `when: contained_availability_group` And test functionality.